prog/vm/host_nexus.rb

```diff
@@ -245,10 +245,14 @@ def os_supports_slices?(os_version)
     total_hugepages = Integer(total_hugepages_match.captures.first)
     free_hugepages = Integer(free_hugepages_match.captures.first)
 
-    spdk_hugepages = vm_host.spdk_installations.sum { |i| i.hugepages }
+    spdk_hugepages = vm_host.spdk_installations.sum(&:hugepages)
     fail "Used hugepages exceed SPDK hugepages" unless total_hugepages - free_hugepages <= spdk_hugepages
 
-    total_vm_mem_gib = vm_host.vms.sum { |vm| vm.memory_gib }
+    total_vm_mem_gib = if vm_host.accepts_slices
```

Comment:
Do we ever have a host that has some VMs in slices and some not in slices? If so, does this code need adjustment to handle that case? If not, this seems fine.